florist/api/db/entities.py

```diff
@@ -0,0 +1,33 @@
+"""Definitions for the MongoDB database entities."""
+import uuid
+from typing import Optional
+
+from pydantic import BaseModel, Field
+from typing_extensions import Annotated
+
+from florist.api.servers.common import Model
+
+
+JOB_DATABASE_NAME = "job"
+
+
+class Job(BaseModel):
+    """Define the Job DB entity."""
+
+    id: str = Field(default_factory=uuid.uuid4, alias="_id")
+    model: Optional[Annotated[Model, Field(...)]]
+    redis_host: Optional[Annotated[str, Field(...)]]
+    redis_port: Optional[Annotated[str, Field(...)]]
+
+    class Config:
```

Comment:
It looks as though though the pydantic V1 behavior to create a class called Config in the namespace of the parent BaseModel subclass is now deprecated [link](https://docs.pydantic.dev/dev/migration/#changes-to-dataclasses). I think the preferred way to specify for config for models is highlighted in this [blog](https://www.mongodb.com/developer/languages/python/python-quickstart-fastapi/)

Reply:
Ah interesting, thanks for pointing that out!

Reply:
I just found out we are using pydantic v1, it's a requirement of pycyclops. Will leave as is until it's migrated.

Reply:
Sounds good!